### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by NuhuWambali** [community/community#67494] [Programming Help] [answered]
### Body

i got this error whenever i tried to push my projects to github..anyone knows how to solve...

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by NuhuWambali** [comment, marked as answer]
ive contacted github support team and they help me to solve this issue

**Reply by atulsingh369** [reply]
how did you contacted them

**Comment by Rashid2021Kalungi** [comment]
How boss
